app/templates/buckets/details.html

```diff
@@ -30,7 +30,7 @@ <h2 class="heading-medium">Users with admin access</h2>
           <td class="align-right">
             {% if bucket.has_admin(current_user) or current_user.is_superuser %}
               <!-- Button to change access level -->
-              <form action="{{ url_for('users3buckets.update', { id: users3bucket.id }) }}" method="post" class="inline-form clearfix">
+              <form action="{{ url_for('users3buckets.update', { id: users3bucket.s3bucket.id }) }}" method="post" class="inline-form clearfix">
```

Comment:
Watch out @axemonkey - this was indeed correct 😱 - in this case you're updating an `UserS3Bucket` so the ID needs to be the one of the `UserS3Bucket`, not of the `S3Bucket`.